metrics-tracing-context/src/label_filter.rs

```diff
@@ -2,13 +2,20 @@
 
 use std::collections::HashSet;
 
-use metrics::Label;
+use metrics::{KeyName, Label};
 
 /// [`LabelFilter`] trait encapsulates the ability to filter labels, i.e.
 /// determining whether a particular span field should be included as a label or not.
 pub trait LabelFilter {
-    /// Returns `true` if the passed label should be included in the key.
+    /// Returns `true` if the passed `label` should be included in the key.
     fn should_include_label(&self, label: &Label) -> bool;
+
+    /// Returns `true` if the passed `label` of the metric named `name` should
+    /// be included in the key.
+    fn should_include_label_by_name(&self, name: &KeyName, label: &Label) -> bool {
+        let _ = name;
+        self.should_include_label(label)
+    }
```

Comment:
Thinking about it, I'd say just stick with only `should_include_label` but do the breaking change of providing `&KeyName`. It feels a little bit simpler to me than having people need to implement _both_ methods if they really only care about overriding `should_include_label_by_name`.

Reply:
All right, done.